paymentsheet/src/test/java/com/stripe/android/paymentelement/confirmation/PaymentElementConfirmationTestActivity.kt

```diff
@@ -41,7 +42,7 @@ import javax.inject.Inject
 import javax.inject.Named
 import javax.inject.Singleton
 
-internal class ConfirmationActivity : AppCompatActivity() {
+internal class PaymentElementConfirmationTestActivity : AppCompatActivity() {
```

Comment:
We would build a separate activity for each confirmation module type and test their behavior in this manner
- `DefaultConfirmationTestActivity`
- `PaymentElementConfirmationTestActivity`
- `ExtendedPaymentElementConfirmationTestActivity`